libpod/container_internal_linux.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/containers/libpod/pkg/lookup"
 	"github.com/containers/libpod/pkg/resolvconf"
 	"github.com/containers/libpod/pkg/rootless"
-	"github.com/cyphar/filepath-securejoin"
+	securejoin "github.com/cyphar/filepath-securejoin"
```

Comment:
did you mean to add 'securejoin' here?  It's fine, but I ask as I don't see it in other changes below.

Reply:
VSCode applied the change automatically. It seems that this package is always referenced by `securejoin.ABC()`, so the change seemed reasonable to me as it avoids confusion about `filepath-securejoin` vs `securejoin`.